src/course-home/data/__snapshots__/redux.test.js.snap

```diff
@@ -444,6 +445,7 @@ Object {
               "effortTime": 15,
               "icon": null,
               "id": "block-v1:edX+DemoX+Demo_Course+type@sequential+block@bcdabcdabcdabcdabcdabcdabcdabcd1",
+              "optional": undefined,
```

Comment:
Should these be `undefined` instead of `true`/`false`?

Reply:
I'm not sure, there were plenty of other undefined values for other fields, so i just left it like this.